pkgs/development/compilers/tinycc/default.nix

```
@@ -1,4 +1,11 @@
-{ stdenv, lib, fetchFromRepoOrCz, perl, texinfo }:
+{ stdenv, lib
+, fetchFromRepoOrCz
+, darwin
+, perl
+, texinfo
+, which
+, xcbuild
+}:
 with lib;
```

```suggestion
with stdenv.lib;
```

I am not a fan of using with on the entire expression. This should be refactored if possible.

---

Yeah I wasn't sure to refactor that since it was the original expression. I agree.